scripts/influxdb.service

```diff
@@ -12,7 +12,7 @@ LimitNOFILE=65536
 Environment='STDOUT=/dev/null'
 Environment='STDERR=/var/log/influxdb/influxd.log'
 EnvironmentFile=-/etc/default/influxdb
-ExecStart=/bin/sh -c "/usr/bin/influxd -config /etc/influxdb/influxdb.conf ${INFLUXD_OPTS} >> ${STDOUT} 2>> ${STDERR}"
+ExecStart=/bin/sh -c "/usr/bin/influxd -config /etc/influxdb/influxdb.conf ${INFLUXD_OPTS} >> ${STDOUT} 2>>${STDERR}"
```

Comment:
It is an issue with systemd and ```stderr``` redirecting were not getting created on our Debian.

Try this:

```
$ echo hi>&2
hi
$ echo hi> &2
bash: syntax error near unexpected token `&'
```

it's bullet proof w/o the space, and yes first space should be remove too